spec/requests/api/v1/conversations/index_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Conversation', type: :request do
+  let(:user) { create(:user) }
+  let(:receivers) { create_list(:user, 4) }
+  let(:topic) { create(:topic) }
+  let(:conversations_response) { json['conversations'] }
+  let!(:conversations) { ConversationService::Creator.new(sender: user, receivers: receivers, topic: topic).call }
```

Comment:
We should use the factory to create the conversations, I would suggest to avoid calling services here.

Services implementation can change, but the goal of this tests is to get the `index` and verify that we can access the data. We shouldn't care if the create service changes, the index should work anyway